pic-api/src/main/kotlin/com/mashup/pic/config/SwaggerConfig.kt

```diff
@@ -23,39 +30,84 @@ import org.springframework.context.annotation.Configuration
-                        .scheme("bearer")
-                        .bearerFormat("JWT"),
-                )
-
         return OpenAPI()
-            .addServersItem(Server().url(serverHostUrl))
-            .addSecurityItem(securityRequirement)
-            .components(components)
+            .addSecurityItem(SecurityRequirement().addList("Authorization"))
+            .components(
+                Components()
+                    .addSecuritySchemes(
+                        "Authorization",
+                        SecurityScheme()
+                            .name("Authorization")
+                            .type(SecurityScheme.Type.HTTP)
+                            .`in`(SecurityScheme.In.HEADER)
+                            .scheme("Bearer")
+                            .bearerFormat("JWT"),
+                    ),
+            )
     }
 
     @Bean
-    fun server(): GroupedOpenApi {
-        return GroupedOpenApi.builder()
-            .group("pic-api")
-            .pathsToExclude("/health")
-            .packagesToScan(ROOT_PACKAGE)
-            .build()
+    fun customiseResponses(): OpenApiCustomizer {
+        return OpenApiCustomizer { openApi ->
+            openApi.paths.orEmpty().values.forEach { pathItem ->
+                pathItem.readOperations().forEach { operation ->
+                    val apiResponses = operation.responses
+                    addStandardResponses(apiResponses)
+                }
+            }
+        }
+    }
+
+    private fun addStandardResponses(apiResponses: ApiResponses) {
+        apiResponses.addApiResponse("400", createStandardResponse("400", "Bad Request"))
+        apiResponses.addApiResponse("403", createStandardResponse("403", "Forbidden"))
+        apiResponses.addApiResponse("405", createStandardResponse("405", "Method Not Allowed"))
+        apiResponses.addApiResponse("500", createStandardResponse("500", "Internal Server Error"))
+    }
+
+    private fun createStandardResponse(
+        code: String,
+        description: String,
+    ): ApiResponse {
+        val exceptionType = getExceptionTypeFromCode(code)
+        val errorResponse =
+            exceptionType?.let {
+                ErrorResponse(code = it.errorCode, message = it.message)
+            }
+        val picApiResponse =
+            PicApiResponse.fail(
+                exceptionType = exceptionType ?: PicExceptionType.ARGUMENT_NOT_VALID,
+                message = errorResponse?.message,
+            )
+
+        val exampleContent = objectMapper.writeValueAsString(picApiResponse)
+        val example = Example().apply { value = exampleContent }
+        val mediaType =
+            MediaType().apply {
+                addExamples("example", example)
+                schema = Schema<Any>()
+            }
+        val content =
+            Content().apply {
+                addMediaType(org.springframework.http.MediaType.APPLICATION_JSON_VALUE, mediaType)
+            }
```

Comment:
```suggestion
        val content = Content().apply {
                addMediaType(org.springframework.http.MediaType.APPLICATION_JSON_VALUE, mediaType)
            }
```
다 밀려서 안예쁘네요. 아마 한 줄에 다 적어서 format 하다가 전부 separate 된거 같은데 한번 더 확인부탁해요